git_hooks/post-checkout

```diff
@@ -2,4 +2,4 @@
 #
 # Writes git revision into git_revision file
 #
-git show-ref refs/heads/master | cut -d " " -f 1  > git_revision
+git rev-parse HEAD | cut -d " " -f 1  > git_revision
```

Comment:
> didn't care about the best way to do it.

Getting a feel here still... so do you want to care now, or should I just remove the affected cuts with one left in in case someone has an old build installed? :question:  *(e.g. [here](https://github.com/Martii/qupzilla/blob/b1258085ad9d5febb6f6708d4eac19ce89698a15/scripts/getrevision.sh#L17) and [here](https://github.com/Martii/qupzilla/blob/b1258085ad9d5febb6f6708d4eac19ce89698a15/scripts/getrevision.sh#L8))*